smoke-tests/src/test/java/com/splunk/opentelemetry/WebSphereSmokeTest.java

```diff
@@ -28,13 +28,13 @@
 public class WebSphereSmokeTest extends AppServerTest {
 
   public static final ExpectedServerAttributes WEBSPHERE8_SERVER_ATTRIBUTES =
-      new WebSphereAttributes("8.5.5.20");
+      new WebSphereAttributes("8.5.5.19");
```

Comment:
Why older?

Reply:
this is because we rely on images build in upstream repo and have to use the same versions. I think at some point there was an issue with `8.5.5.20` image so it got downgraded to `8.5.5.19` to get the build passing. if you want to change this I'd suggest you first change it in upstream and when that change is merged change it here also

Reply:
Cool, thanks for clarifying.